mods/cnc/rules/ai.yaml

```diff
@@ -307,3 +307,5 @@ Player:
 			orca: 10
 		UnitLimits:
 			harv: 8
+	PowerDownBotMoudle:
```

Comment:
TD doesn't support powering down structures.

Reply:
Thanks for mentioning.

Reply:
fixed